aws/resource_aws_eip.go

```diff
@@ -70,11 +70,23 @@ func resourceAwsEip() *schema.Resource {
 				Computed: true,
 			},
 
+			"public_dns": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
 			"private_ip": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
 
+			"private_dns": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
`Optional: true` should be removed here as operators cannot specify this argument.